compiler/luci/pass/src/ExpandBroadcastConstPass.cpp

```diff
@@ -47,13 +47,18 @@ luci::CircleConst *create_expanded_constant(luci::CircleConst *node, luci::Circl
     return nullptr;
   }
 
+  if (successor->rank() == 1 || successor->rank() > 4)
+  {
+    WARN(l) << "NYI: Only 2D/3D/4D tensor broadcast removal is supported";
+    return nullptr;
+  }
+
```

Comment:
@seanshpark This part is updated. :-D The condition check was done as assert before, but now it returns nullptr.